Source/Scene/modelMaterialsCommon.js

```diff
@@ -349,6 +349,16 @@ define([
             vertexShader += 'attribute vec4 a_joint;\n';
             vertexShader += 'attribute vec4 a_weight;\n';
         }
+        
+        if (options.addBatchIdToGeneratedShaders)
+        {
+            techniqueAttributes.a_batchId = 'batchId';
+            techniqueParameters.batchId = {
+                semantic: 'BATCHID',
+                type: WebGLConstants.UNSIGNED_SHORT
```

Comment:
Thanks @lexaknyazev for the update.

@JDepooter the type can be `WebGLConstants.FLOAT`. I also updated the 3D Tiles spec: https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/154